menuinst/win32.py

```diff
@@ -115,6 +115,20 @@ def quoted(s):
         return s
 
 
+def ensure_pad(name, pad="_"):
```

Comment:
I added this function from conda because I don't think `quoted()` above does the right thing.  I definitely don't understand why `u'/' in s` gets quoted.